using-gh-actions-and-okteto-cloud-to-preview-pr-changes.md

```diff
@@ -0,0 +1,185 @@
+
+
+## Creating A GitHub Action Workflow
+
+GitHub Actions are created and managed from the **Actions** tab in a repository. Create a new workflow from the Actions tab:
+
+![Create A New Workflow](https://res.cloudinary.com/laisi/image/upload/v1615041975/emgx1lf6c9qe0snrxq9a.png)
+
+There are a number of available templates in the GitHub Actions page. However, click on the **set up a workflow yourself** link. This opens a new page with a default workflow, let's rewrite the workflow file:
+
+Start by defining the name of the workflow, and the event which should trigger the execution of the workflow file:
+
+```yaml
+name: CRUD App Preview Deployment
+
+on: 
+  pull_request:
+    branches:
+      - main
+
+```
+
+In the code block above, we have set the name of the workflow, and the event to trigger the workflow. The workflow file is executed when a pull request is made to the `main` branch. Next, let's define the series of tasks to be executed. These tasks are called **jobs** in GitHub Actions.
+
+Add the following below the existing code:
+
+```yaml
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@master
+
+    - name: Login
+      uses: okteto/login@master
+      with:
+        token: ${{ secrets.OKTETO_TOKEN }}
+
+    - name: Create namespace
+      uses: okteto/create-namespace@master
+      with:
+        namespace: pr-${{ github.event.number }}-namespace
+   
+    - name: Deploy application
+      uses: okteto/deploy-stack@master
+      with:
+        build: "true"
+        namespace: pr-${{ github.event.number }}-namespace
+
+    - name: add comment to PR
+      uses: unsplash/comment-on-pr@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        msg: "Preview environment available at https://fastapi-pr-${{ github.event.number }}-namespace.cloud.okteto.net"  
+```
+
+In the code block above, we defined the job `preview` which runs on the `ubuntu-latest` environment. The subtasks to be executed under this job is defined under the `steps` heading. Let's go over these subtasks.
+
+Replace `namespace` in the code block above with your Okteto default namespace.
+
+### Subtasks
+
+In our `preview` job, we have five tasks to be executed. The first task checks out the repository using the `actions/checkout@master` actions, so that the repository is accessible by the entire workflow. 
+
+The second task logs in into your Okteto account as a prerequisite to creating a new namespace where the preview environment will be deployed. This task makes use of a token `secrets.OKTETO_TOKEN` which is your Okteto token. In the next section, you'll learn how to add your okteto token to your repository.
```

Comment:
Instead of saying `first`, `second`, `third`, I think it would be easier to read if we do a list of the steps, with the name of the step as the first part:

1. `checkout`:  This step checks out the repository....
2. `Login`:  This step logs in into your Okteto account...
3. ....
4. ...

Reply:
Fixed